README.md

```diff
@@ -182,6 +182,7 @@ vars:
   hubspot__contact_pass_through_columns:
     - name:           "wow_i_can_add_all_my_custom_fields"
       alias:          "best_field"
+      use_property_label: true
```

Comment:
I feel we should not include this in the instructions until we explain it in the following section.